.github/workflows/wasm-build.yaml

```
@@ -122,51 +163,73 @@ jobs:
       run: |
-        ATOMVM_JS=AtomVM-node-${{ github.ref_name }}.js
-        mv AtomVM.js "${ATOMVM_JS}"
+        ATOMVM_JS=AtomVM-node-${{ github.ref_name }}.mjs
+        mv AtomVM.mjs "${ATOMVM_JS}"
         sha256sum "${ATOMVM_JS}" > "${ATOMVM_JS}.sha256"
         ATOMVM_WASM=AtomVM-node-${{ github.ref_name }}.wasm
         mv AtomVM.wasm "${ATOMVM_WASM}"
         sha256sum "${ATOMVM_WASM}" > "${ATOMVM_WASM}.sha256"
 
     - name: "Release (node)"
       uses: softprops/action-gh-release@v1
```

actionlint (1.7.12)

[error] the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)